primitives/header-chain/src/lib.rs

```diff
@@ -138,6 +138,22 @@ impl<H: HeaderT> AncestryChecker<H, Vec<H>> for LinearAncestryChecker {
 	}
 }
 
+/// Find header digest that schedules next GRANDPA authorities set.
+pub fn find_grandpa_authorities_scheduled_change<H: HeaderT>(
```

Comment:
I think it's required - it lays within something that is called `bp_header_chain`, which doesn't mention GRANDPA. There may be (and are) other changes in header digest.

Reply:
The only other change (that I'm aware of) is a force change.

I prefer the shorter name, but I won't you fight over this :)

Reply:
> The only other change (that I'm aware of) is a force change.

There's also at least BABE and Aura, which have their own consensus changes with digests. Don't know about other consensus algorithms, but pretty sure that every will have something like that (if they're sealing blocks with signatures).

> The header_chain module is clearly tied to GRANDPA by using `sp_finality_grandpa`

Looking at crate dependencies/module usages to understand what the function does isn't a super-optimal workflow imo. When you're exposing public function, it should be clear what it does just by looking at its path+name. Otherwise you'll need all those extra steps (looking into deps, usages) to understand what's happening. And since header-chain is neutral name, it'll be confusing to provide something that returns anything other than grandpa-authorities-scheduled-change imo. This name may be shortened if we'll change crate name to something grandpa-specific.